SQF/dayz_code/stringtable.xml

```diff
@@ -13403,6 +13412,24 @@
 			<French>Votre humanité est trop %1 ce commerçant refuse de vous parler.</French>
 			<Czech>Tvá lidskost je příliš %1, tento obchodník s tebou odmítá mluvit.</Czech>
 		</Key>
+		<Key ID="STR_EPOCH_ACTIONS_HUMANITY_HIGH">
+			<English>high</English>
+			<German>high</German> -->
```

Comment:
You forgot to remove the closing comment <code>--></code> tag on this line and line 13419. Everything else is fine. Fix that and I'll merge it. 

BTW in the future you may want to put simple strings like "Book", "Lightbulb", "high", "low", etc. into Google Translate for the other languages. It is usually correct for single words.